src/ray/protobuf/common.proto

```diff
@@ -195,7 +220,7 @@ message CoreWorkerStats {
   // Number of object ids in local scope.
   int32 num_object_ids_in_scope = 3;
   // Function descriptor of the currently executing task.
-  repeated bytes current_task_func_desc = 4;
+  string current_task_func_desc = 4;
```

Comment:
Why it's changed to type of `string` instead of type of `FunctionDescriptor`?

Reply:
This is only used for showing the fd on dashboard. Using string is fine. Can you also update the comment above:
`String representation of the function descriptor of the currently executing task`